src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -139,8 +139,10 @@ public static void NonExistentAction_Throws_ActionNotSupportedException()
     // The client should throw a TimeoutException
     [Fact]
     [OuterLoop]
+    [ActiveIssue(3464)]// Issue is in the corefx repo
     public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutException()
```

Comment:
I think we can make this issue #273 because it is in our repo.  #273 should probably be updated to refer to the CoreFx issue.